content/projects.md

```diff
@@ -0,0 +1,21 @@
+# Projects
```

Comment:
What do you think of linking in projects(subpages) the flow-dps, flow-insights and the projects that optakt have on hands?

Reply:
This is what is planned in the last section of this page, yes. I was thinking we'd have something like

```md
## Documentation

* Flow DPS
	* Intro
	* Architecture
	* Binaries
		* flow-dps-indexer
		* flow-dps-live
		* ...
* Flow Rosetta
* etc.
```

Reply:
Yeah, only two projects, but when we start to get bigger will be good to automate it, otherwise it can get outdated very quickly.